sda-orch/templates/networkpolicies.yaml

```diff
@@ -23,45 +23,69 @@ spec:
   - to:
     - namespaceSelector:
         matchLabels:
-          name: kube-system
+          networkpolicy: kube-system
     ports:
     - protocol: UDP
       port: 53
 ---
 kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
 metadata:
-  name: sda-orch
+  name: db-policy
   namespace: {{ .Release.Namespace }}
 spec:
   podSelector:
     matchLabels:
-      app: {{ template "sda.fullname" . }}
+      {{ .Values.networkPolicy.databaseNamespace }}: allowtraffic
```

Comment:
This determines what pods the policy is applied to, so it should be the same label as in the po. So `app: {{ template "sda.fullname" . }}` is probbably correct.

Reply:
It would be simpler to allow all egress for orch. But would that be too open?

Reply:
copied this from sda-svc which seems to be the same :thinking: should we change there ?

Reply:
My bad. Nothing wrong with this after all